vulcan/lib/client/jsx/contexts/session_sync.tsx

```diff
@@ -72,7 +73,7 @@ export function useSessionSync(
 
     if (!post &&
         hasNoRunningSteps(state.current.status) &&
-        lastCompletedPollingRequest === state.current.session.inputs &&
+        _.isEqual(lastCompletedPollingRequest, state.current.session.inputs) &&
```

Comment:
UI was constantly polling once we introduced a Hash-like object as an input value because this equality was always `false`, so had to convert this to do a deep equality check.